db/re/item_db.txt

```diff
@@ -3923,7 +3923,7 @@
 5734,E_Magestic_Goat_C,E Magestic Goat C,4,2,,0,,5,,0,0xFFFFFFFE,63,2,256,,0,0,41,{ bonus bStr,1; },{},{}
 5735,E_Ribbon_Green,Green Ribbon,4,800,,100,,1,,0,0xFFFFFFFF,63,0,256,,0,1,244,{ bonus bMdef,3; },{},{}
 5736,EF_Whisper_Mask,Whisper Mask,4,20,,0,,0,,0,0xFFFFFFFF,63,2,769,,0,0,321,{ bonus bAgi,3; bonus2 bSubEle,Ele_Ghost,-10; },{},{}
-5737,Cactus_Hat,Cactus Hat,4,20,,300,,1,,0,0xFFFFFFFF,63,2,256,,1,0,615,{ bonus3 bAddMonsterDropItem,952,RC_Plant,500; },{},{}
+5737,Cactus_Hat,Cactus Hat,4,20,,300,,1,,0,0xFFFFFFFF,63,2,256,,1,0,615,{ skill "SM_PROVOKE",3; },{},{ sc_end SM_PROVOKE; }
```

Comment:
I actually found an Aegis source that has the Cactus Needle drop script to instead of this. thRO item description also only has the Cactus Needle part. Could you confirm this isn't an iRO customization?

Anyways, provoke isn't a self skill. Why end the status on the equipping player?

Reply:
kRO doesn't actualy have this in their itemInfo.lub, so i doubt they even have it ingame.
But yea, seems like an iRO customization, sry about that.
Will revert it.